src/components/views/Dashboard.jsx

```diff
@@ -46,21 +47,40 @@ export default props => {
     let maybeGraphDescription = null;
 
     if (props.description) {
-        maybeDescription = (
-            <p id="dashboard-description">{props.description}</p>
-        );
+        const multipleParagraphs = Array.isArray(props.description);
+        const markdownParser = markdownIt('zero').enable(['emphasis']);
+
+        if (multipleParagraphs) {
+            maybeDescription = (
+                <div id="dashboard-description">
+                    {props.description.map(paragraph => (
+                        <p key={paragraph} dangerouslySetInnerHTML={
```

Comment:
seems like a really long key, if the order of these in the DOM won't change it should be alright to key the index

Reply:
Good call. I had read that indices should not be used as keys, but upon further reading this seems to be a perfect exception to the rule. 👍